internal/condition/condition.go

```diff
@@ -0,0 +1,35 @@
+package condition
+
+/*
```

Comment:
Maybe add a comment here about what this is for? The naming is a big vague.

Reply:
Literally none of the Go files have package or file-level comments at the top.
I say this only because I'm wondering if it should go _before_ the `package` statement, or after?

Reply:
It should be before.
https://blog.golang.org/godoc

Reply:
Thanks. It's been a while since I was diligent about Go program documentation.
I went all-out on one project a couple years ago.